text/0012-CLI-commands.md

```diff
@@ -0,0 +1,462 @@
+## qri export
+
+Copy datasets from qri to your local filesystem
+
+### Synopsis
+
+
+Export gets datasets out of qri. By default it exports only the dataset body. 
+
+To export to a specific directory, use the --output flag.
+
+If you want an empty dataset that can be filled in with details to create a
+new dataset, use --blank.
+
+```
+qri export [flags] <dataset_reference>
+```
+### Examples
+
+```
+  # export dataset
+  qri export me/annual_pop
+
+  # export without the body of the dataset
+  qri export --no-body me/annual_pop
+
+  # export to a specific directory
+  qri export -o ~/new_directory me/annual_pop
+```
+
+### Options
+
+```
+      --blank                export a blank dataset YAML file, overrides all other flags except output
+      --body-format string   format for dataset body. default is the original data format. options: json, csv, cbor
+  -f, --format string        format for all exported files, except for body. yaml is the default format. options: yaml, json (default "yaml")
+  -h, --help                 help for export
+  -b, --no-body              don't include dataset body in export
+  -o, --output string        path to write to, default is current directory
+  -d, --peer-dir             export to a peer name namespaced directory
+```
+
+
+<a id="qri_get"></a>
+## qri get
+
+Get elements of qri datasets
+
+### Synopsis
+
+Get the qri dataset (except for the body). You can also get portions of 
+the dataset: meta, structure, viz, transform, and commit. To narrow down
+further to specific fields in each section, use dot notation. The get 
+command prints to the console in yaml format, by default.
+
+You can get pertinent information on multiple datasets at the same time
+by supplying more than one dataset reference.
+
+Check out the [dataset documentation](https://qri.io/docs/reference/dataset/) to learn about each section of the 
+dataset and its fields.
+
+```
+qri get [flags] <dataset_reference> <dataset_reference2> ...
+```
+
+### Examples
+
+```
+  # print the entire dataset to the console
+  qri get me/annual_pop
+
+  # print the meta to the console
+  qri get meta me/annual_pop
+
+  # print the dataset body size to the console
+  qri get structure.length me/annual_pop
+
+  # print the dataset body size for two different datasets
+  qri get structure.length me/annual_pop me/annual_gdp
+```
+
+### Options
+
+```
+      --concise         print output without indentation, only applies to json format
+  -f, --format string   set output format [json, yaml] (default "yaml")
+  -h, --help            help for get
+```
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+<!-- - Why is this design the best in the space of possible designs?
+- What other designs have been considered and what is the rationale for not choosing them?
+- What is the impact of not doing this? -->
+
+# Prior art
+[prior-art]: #prior-art
```

Comment:
I think it'd be great to list kubernetes here & link to the github repo (possibly even link to the package that includes the `NewKubernetesCommand` stuff?)